src/System.Private.Reflection.Core/src/System/Reflection/Runtime/TypeInfos/RuntimeTypeInfo.cs

```diff
@@ -356,7 +356,7 @@ public sealed override MemberTypes MemberType
             }
         }
 
-        public sealed override int MetadataToken
+        public override int MetadataToken
```

Comment:
This one can probably remain unsealed as they're so many subclasses. But please add a comment similar to this one so we know it was an intentional decision to leave unsealed.

        //
        // Left unsealed as array types must override this.
        //
        public override int GetArrayRank()